packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.2.1",
+  "version": "6.9.0",
```

Comment:
I'm thinking about syncing the version just for the sake of aligning the Babel versions.
What do you think?